docker/docker-compose.ltc.yml

```diff
@@ -0,0 +1,54 @@
+version: '2'
+services:
+    # ltc is an image of litecoin node which used as base image for ltcd and
+    # ltcctl. The environment variables default values determined on stage of
+    # container start within starting script.
+    ltcd:
+      image: ltcd
+      container_name: ltcd
+      build:
+        context: ltcd/
+      volumes:
+        - shared:/rpc
+        - litecoin:/data
+      environment:
+        - RPCUSER
+        - RPCPASS
+        - NETWORK
+        - DEBUG
+        - MINING_ADDRESS
+      entrypoint: ["./start-ltcd.sh"]
+
+    lnd:
+      image: lnd
+      build:
+        context: ../
+        dockerfile: docker/lnd/Dockerfile
+      environment:
+        - RPCUSER
+        - RPCPASS
+        - NETWORK
+        - CHAIN
+        - DEBUG
+      volumes:
+        - shared:/rpc
+        - lnd:/root/.lnd
+      entrypoint: ["./start-lnd.sh"]
+      links:
+        - "ltcd:blockchain"   
```

Comment:
nit: there are a few trailing spaces on this line

Reply:
fixed